[DOCS] 8.1.0 Release Notes (#1633)
elastic/security-docs: docs/release-notes/8.1.asciidoc

[[release-notes-header-8.1.0]]
== 8.1

[discrete]
[[release-notes-8.1.0]]
=== 8.1.0

[discrete]
[[breaking-changes-8.1.0]]

> ==== Breaking Changes

Reviewer: ```suggestion
==== Breaking changes
```

// tag::breaking-changes[]
// NOTE: The breaking-changes tagged regions are re-used in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
:pull: https://github.com/elastic/kibana/pull/
There are no breaking changes in 8.1.0.
// end::breaking-changes[]

[discrete]
[[features-8.1.0]]
==== Features

> * Improves copy for the privilege check on the Endpoint page ({pull}124118[#124118]).

Reviewer 2: ```suggestion
* Improves copy for the privilege check on the Endpoints page ({pull}124118[#124118]).
```

> * Adds the Technical preview toggle above the Rules table which, when enabled, allows users to sort on all rule management columns ({pull}119611[#119611]).

Reviewer 3: ```suggestion
* Adds a Technical preview toggle above the Rules table which, when enabled, allows users to sort on all rule management columns ({pull}119611[#119611]).
```

> * Improves copy for the privilege check on the Endpoints page ({pull}124118[#124118]).

Reviewer 3: maybe put this last? It's relatively minor

* Introduces the ability to  bulk edit rule index patterns and tags ({pull}122635[#122635]).
* Expands Endpoint per-policy artifact assignment to include endpoint event filters and host isolation IP exceptions ({pull}121879[#121879], {pull}121632[#121632]).

> * Adds the rule execution UUID field to alerts. In addition, the `kibana.alert.rule.execution.uuid` is now part of the alert data schema and can be found in the field browser for the Alert's table.({pull}113058[#113058]).

Reviewer 2: We should probably include the actual rule UUID field here and wrap it in `code text`. 
```suggestion
* Adds the rule execution UUID field to alerts. In addition, the `kibana.alert.rule.execution.uuid` field is now part of the alert data schema and can be found in the field browser in the Alerts table.({pull}113058[#113058]).
```

Author: Resolving because added the recommended edits.

* Introduces case metrics that summarize alert information and response times ({pull}121336[#121336]).

[discrete]
[[bug-fixes-8.1.0]]
==== Bug fixes and enhancements
* Improves the performance of indicator match rules ({pull}123882[#123882], {pull}123677[#123677]).
* Changes the default indicator index query of custom and prebuilt indicator match rules to `@timestamp >= "now-30d/d"` ({pull}123590[#123590]).
* Improves the exceptions interface by replacing the exceptions modal with a flyout ({pull}123408[#123408]).
* Alert details flyout enhancements:
** Shows different “highlighted fields” in an alert’s details flyout based on its type, category, and code ({pull}123239[#123239]).
** Adds overview cards with key data to the alert details flyout ({pull}120347[#120347]).
** Allows you to aggregate alert data based on any aggregatable ECS field instead of just 10 preset options ({pull}120610[#120610]).

> * Speeds up the rules table by implementing an in-memory rules cache ({pull}119611[#119611]).

Reviewer: Currently this is an optional technical preview feature -- should we mention this?

This feature also allows the user to sort the rules table on more columns, which is more of an outright new feature so we should probably add this to the Features section above.

Author: @pedro we could use your input here! It seems that #119611 added quite a bit in terms of enhancements to the **Rules** tab and **Rule Monitoring** tab on the Rules page. Does the current summary fully represent that? If not, what changes do you recommend? Also, do you have a preference for where this PR resides in the release notes? The actual release note label on the PR categorizes it as an enhancement, which is why we placed it here. However, if you'd prefer that these changes are more visible, and consider the changes significant enough that they warrant the "new feature" label, we can move it to the New Features section.

Reviewer 4: Hi @nastasha-solomon, I would definitely favor moving this to the `Features` section because it's a highly requested enhancement.

As far as summarizing it, I would only highlight that the experimental feature allows users to to sort on all rule management columns. We don't need to highlight the performance enhancements described in the PR because there are trade offs with performance.

> ** Allows you to aggregate alert data based on any aggregatable ECS field instead of just 10 preset options ({pull}120610[#120610]).

Reviewer 2: Just a thought -- I'm wondering if customers are going to know what `aggregatable ECS field` is supposed to really mean...

Author: @jmikell821 @benironside and I discussed alternatives for `aggregatable ECS field` and settled on not qualifying the type of ECS field that the **Count** and **Trend** tables use. In this particular case, specifying the type of ECS fields that the tables use might be more confusing than helpful. What do you think about changing that part of the summary to something more general like "a larger selection of fields"?

Author: Also @benironside the summary for #120610 might need to have its own line because it moreso affects the Trend and Count tables on the Alerts page and Overview page.

> * Speeds up the rules table by implementing an in-memory rules cache ({pull}119611[#119611]).

Reviewer 2: ```suggestion
* Improves speed performance for the Rules table by implementing an in-memory rules cache ({pull}119611[#119611]).
```

> * Enrich threshold-related alert data from correct fields ({pull}125376[#125376]).

Reviewer 2: ```suggestion
* Enriches threshold-related alert data from correct fields ({pull}125376[#125376]).
```
